python/lsst/afw/display/interface.py

```diff
@@ -230,17 +257,22 @@ def __str__(self):
     # Handle Displays, including the default one (the frame to use when a user specifies None)
     #
     @staticmethod
-    def setDefaultBackend(backend):
+    def setDefaultBackend(backend, *args, **kwargs):
+        Display._args = args
+        Display._kwargs = kwargs
         try:
-            _makeDisplayImpl(None, backend)
+            _makeDisplayImpl(None, backend, *Display._args, **Display._kwargs)
         except Exception as e:
             raise RuntimeError("Unable to set backend to %s: \"%s\"" % (backend, e))
 
         Display._defaultBackend = backend
 
     @staticmethod
-    def getDefaultBackend():
-        return Display._defaultBackend
+    def getDefaultBackend(optional_args=False):
```

Comment:
These are excellent points about the deficiencies in the interface. I agree that the implemented solution is brittle. 

Logically, the host and port belong to the `Display`. A user can specify host and port for each Firefly Display along with the frame number, and optionally, a channel name. 

After some discussion with @xiuqin , we are thinking it is better to not implement this change to `setDefaultBackend` and to just tell people to use the method that already works, using the `Display` constructor.  This pull request just provides an alternative path to defining displays, that would probably be better left to a future redesign of the afwDisplay interface.